sandbox/handlers/get_message.js

```diff
@@ -1,18 +1,19 @@
 import * as  fs from 'fs'
-import { sendError, write_log } from './utils.js'
+import { sendError, writeLog } from './utils.js'
 
-export async function get_message(req, res, next) {
-  if (req.headers["authorization"] === "banned") {
+export async function getMessage(req, res, next) {
+  if (req.headers.authorization === "banned") {
     sendError(res, 403, "Request rejected because client service ban is in effect");
     next();
     return;
   }
 
-  const messageId = req.params.messageId;
+  const { messageId } = req.params;
 
+  /* eslint-disable-next-line security/detect-non-literal-fs-filename */
```

Comment:
Are we sure we don't want to resolve these rather than just put in an ignore statement? It's trying to highlight to us that one could come along and go `curl http://sandbox.url/messages?messageId=../../../../../some-important-json-file.json`

Maybe this doesn't matter since it's only our sandbox app which is only intended for internal use, but in that case I would suggest we should disable the linter rule entirely rather than just put in an ignore statement?

Reply:
Yes, I didn't think it mattered much since it's only the sandbox app, but that's a good point. I'll disable the rule for sandox.